core/src/main/java/org/gbif/pipelines/core/ws/client/internal/GBIFInternalResponse.java

```diff
@@ -0,0 +1,81 @@
+package org.gbif.pipelines.core.ws.client.internal;
+
+import java.io.Serializable;
+import java.util.List;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * Response structure of the aggregated gbif terms
+ */
+public class GBIFInternalResponse implements Serializable {
```

Comment:
missing hashCode and equals, please make all fields final and add Builder, this will allow a much better approach at the assembling time of this class....and lets change it DatasetInfo or something  similar